packages/sanity/src/config/prepareConfig.ts

```diff
@@ -201,7 +201,8 @@ function getBifurClient(client: SanityClient, auth: AuthStore) {
 
   const url = bifurVersionedClient.getUrl(`/socket/${dataset}`).replace(/^http/, 'ws')
 
-  return fromUrl(url, auth.token ? {token$: auth.token} : {})
+  const options = auth.token ? {token$: auth.token} : {}
+  return fromUrl(url, options)
```

Comment:
Is this change necessary?

Reply:
TypeScript was shouting that token$ is not allowed in literal type. Might just be IntelliJ shouting though, so I can revert if you want.

Reply:
Hmm I think maybe that was a fluke locally; reverting.